components/select_team/select_team.jsx

```diff
@@ -153,7 +153,7 @@ export default class SelectTeam extends React.Component {
                         <label className='control-label'>
                             <FormattedMessage
                                 id='signup_team.no_teams_for_guests'
-                                defaultMessage={'You don\'t belongs to any team and as a guest you can\'t join any team by yourself, you need to be invited or added by someone else.'}
+                                defaultMessage='Your guest account has no channels assigned. Please contact an administrator.'
```

Comment:
The id says that there are no teams for the guest, but the default message says it doesn't have any channels - Is the translation id incorrect?

Reply:
Yes, actually, a guest is member of a team only if is a member of one of the channels in the team. But, here we have an empty list of teams, which means that you don't belongs to any "team" or "channel", but from the guest perspective, they should care more about channels than teams. But I'm ok changing the id to something like `signup_team.guest_without_channels`.